### PR TITLE
Contact Us page Created

### PR DESCRIPTION
**Pull request by ankittripathi31**

### CODE REVIEW

**Review by VarunSAthreya — CHANGES_REQUESTED** [review]
Please send a PR to `Android-Development:development` branch.

**Comment by VarunAthreya** [comment]
I have done the changes please do it from next time.

File: lib/ContactUs/customTextField.dart [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
```
Comment (VarunAthreya) [inline lib/ContactUs/customTextField.dart:1]: This file is not being used anywhere so delete it.

**Review by VarunSAthreya — CHANGES_REQUESTED** [review]
There are a few changes:
1. Remove MyHomePage from home in main.dart and use ContactPage as home.
2. There is an error in Send Message
error1

3. Screen overflow when filling the form.
Screenshot_1603268394

4. customTextField.dart file isn't being used so remove it.

**Comment by ankittripathi31** [comment]
OK i have to put the ContactPage code to main.dart. I'm i correct

**Review by VarunSAthreya — CHANGES_REQUESTED** [review]
Changes:
1. There are many merge conflicts which need to be fixed.
2. Move `contactPage.dart` to the folder `screens`.
3. Delete customTextField.dart

**Pull request closed without merging**